src/components/helpers/assoc-list-item-key.js

```diff
@@ -36,6 +36,7 @@ export default createReactClass({
 
     return (
       <input
+        aria-labelledby={this.props.id ? `${this.props.id}_label` : undefined}
```

Comment:
Hmm, is this accurate? Is it appropriate to have multiple inputs be labeled by a single label? It seems like something like `aria-label="key"` would be more appropriate here? And `aria-labelledby` should apply to the widget as a whole?

Reply:
Yeah I wasn't sure what to do here, I think your idea is better 👍